topics/client-engines.md

```
@@ -60,181 +67,236 @@ The table below shows whether a specific engine supports HTTP/2 and [WebSockets]
+> Ktor provides platform-specific artifacts with suffixes such as `-jvm` or `-js`. For example, `ktor-client-cio-jvm`.
+> Dependency resolution differs by build tool. While Gradle resolves artifacts appropriate for a given platform, Maven
+> doesn't support this capability. This means that for Maven, you need to specify the platform suffix manually.
 >
 {type="note"}
 
-## Create a client with a specified engine {id="create"}
-To create the HTTP client with a specific engine, pass an engine class as an argument to the [HttpClient](https://api.ktor.io/ktor-client/ktor-client-core/io.ktor.client/-http-client/index.html) constructor. For example, you can create a client with the `CIO` engine as follows:
+## Specify an engine {id="create"}
+
+To use a specific engine, pass the engine class to the [
+`HttpClient`](https://api.ktor.io/ktor-client/ktor-client-core/io.ktor.client/-http-client/index.html) constructor. The
```

markdownlint-cli2 (0.17.2)

Spaces inside link text

(MD039, no-space-in-links)